ansible-scylla-node/upgrade/main.yml

```diff
@@ -0,0 +1,305 @@
+---
+
+# ---[ TODO ]-----------------------------------------------
+#    0) Test better Ubuntu/Debian distros for specific version upgrades. Check if apt module should be used or not.
+#    1) Downgrade support
+#    2) Cross-edition upgrade/downgrade
+# ---------------------------------------------------------
+
+# Scylla matrix for supported distros, editions and versions
+- name: Define Scylla support matrix
+  ansible.builtin.set_fact:
```

Comment:
duplicated matrix? 
this asks for a single place where to take it from if possible